diff --git a/dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/DBUniqueFieldValidationStrategy.java b/dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/DBUniqueFieldValidationStrategy.java
index e35079736c4e..674410d9bfe7 100644
--- a/dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/DBUniqueFieldValidationStrategy.java
+++ b/dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/DBUniqueFieldValidationStrategy.java
@@ -7,7 +7,6 @@
 import static com.dotmarketing.util.Constants.DONT_RESPECT_FRONT_END_ROLES;
 import static com.liferay.util.StringPool.BLANK;
 
-import com.dotcms.business.CloseDBIfOpened;
 import com.dotcms.business.WrapInTransaction;
 import com.dotcms.contenttype.business.UniqueFieldValueDuplicatedException;
 import com.dotcms.contenttype.business.uniquefields.UniqueFieldValidationStrategy;
@@ -63,7 +62,6 @@ public DBUniqueFieldValidationStrategy(final  UniqueFieldDataBaseUtil uniqueFiel
 
     @Override
     @WrapInTransaction
-    @CloseDBIfOpened
     public void innerValidate(final Contentlet contentlet, final Field field, final Object fieldValue,
                        final ContentType contentType) throws UniqueFieldValueDuplicatedException, DotDataException, DotSecurityException {
 
@@ -77,7 +75,6 @@ public void innerValidate(final Contentlet contentlet, final Field field, final
 
     @Override
     @WrapInTransaction
-    @CloseDBIfOpened
     public void innerValidateInPreview(final Contentlet contentlet, final Field field, final Object fieldValue,
                                         final ContentType contentType)
             throws UniqueFieldValueDuplicatedException, DotDataException, DotSecurityException {
